Updating Calendar document

jpwalden opened the pull request:
Added how to heal google sync -chapter, added weekly view and made some other updates as well.

jovirkku commented:
If we wanted, line 26 could also tell that the dot colours denote various sub-calendars. The colours can be changed in the "Manage calendars" of the pulley menu.

jovirkku commented on Support/Help_Articles/Calendar/README.md:
```diff
@@ -24,31 +24,43 @@ When your Calendar app opens, you are greeted with the Month view.
+## Weekly View
+
+* You can access the weekly view from the left top corner by selecting the weekly view (icon in the middle).
+* In the weekly view, it is easier to see the summary of the week's events.
```
Even better if you turn the device to the landscape position.

jpwalden replied:
Added comment about landscape

jovirkku commented on Support/Help_Articles/Calendar/README.md:
```diff
@@ -124,10 +136,47 @@ Google calendar and Microsoft Exchange calendar are examples of calendar types,
+The following is a kind of reset of the sync utility. It does not do any harm to the appointments or contacts.
+ 
+## Preparations
+1. You will need the Developer mode and the SSH password. Do as instructed in this help article.
```
Link to the document missing?

jpwalden replied:
Thanks, that was missing -> added.

jovirkku reviewed (changes requested):
The modified parts (which are under review) are almost ok. Little needs be reworked.

If we want to improve the unmodified parts (unmodified by this commit) then there is more to do.

jpwalden commented:
> As I started reading the rich view of the whole doc (not just the changed parts) I noticed the following slight problems - the implementation of the app has changed over the years but the document text not so much...
> 
> If we wanted, line 26 (or line 28) could also tell that the dot colours denote various sub-calendars. The colours can be changed as explained in chapter "Changing calendar colours"
> 
> On the old line 56 or the new line 68, just under "_Creating a new Event in Calendar_", the sentence claims that new events should be created in the Month view. But they can be created in the week and day views, too. All 3 views share the same pulley menu. So, the sentence could be something like: "_When it’s time to mark something important down, pull down “Create new event”_.
> 
> Remove the last part of the sentence, i.e. " _where it shows what you chose for the date or time._" from line 74/86 (end of step 4). Namely, the current OS version has the Accept button in the top right corner. It does not show the selected date or time there.
> 
> Line 77/89: I do not think we have the "read-only" tagging anymore. Do we? If not, the "please note" sentence (2) should be removed. Line 113/125 refers to "read-only", too. I wonder if it is correct. Would Pekka know?

Good observations, following changes made: colours mentioned, fixed the create event part, since that can be indeed done also from weekly and daily view, removed the last part of the sentence in step4. Checked from Pekka that read-only events part is still valid, so didn't remove that.

jpwalden commented on Support/Help_Articles/Calendar/README.md:
```diff
@@ -24,31 +24,43 @@ When your Calendar app opens, you are greeted with the Month view.
+## Day View
 
-At the bottom of the view, you can see events that are coming up if you scroll further down. At the top of the screen, you can see events that are in the past that you can see by scrolling up.
+* You can access the day view from the left top corner by selecting the weekly view (icon in the right).
```
Fixed

jpwalden commented on Support/Help_Articles/Calendar/README.md:
```diff
@@ -124,10 +136,47 @@ Google calendar and Microsoft Exchange calendar are examples of calendar types,
-Birthday events cannot be created, deleted or modified in Calendar because they are obtained through syncing. It is, however, possible to change or delete a person’s birthday by modifying that person’s information in the People application (phonebook). Thereafter the modification made will appear in Calendar, too.
-The sync schedule for your calendars is the same as for the corresponding email service. For Exchange, there are more refined sync options in Settings > Accounts > Exchange.
-You can snooze a calendar alert by flipping the phone (turn it screen downwards) or by pressing the power key shortly.
+* If you set an end date for your calendar entry, the event will end on the date you’ve specified, but the calendar will treat it as a non-stop event. This means that an event will begin the date you specify, start at the time you specify, and then go on uninterrupted until the end date and time you time you’ve specified.
```
That is quite unclear sentence, seems to cause more confusion than clarification, so removed it.

jovirkku reviewed (approved):
This looks ok now. 

[at new line 68, step 5, selecting the calendar: I think it is still so that the calendar once selected for an event cannot later be changed anymore - so if one mistakenly selects a wrong calendar then he should open a new event to force it to another calendar]

jpwalden commented:
> This looks ok now.
> 
> [at new line 68, step 5, selecting the calendar: I think it is still so that the calendar once selected for an event cannot later be changed anymore - so if one mistakenly selects a wrong calendar then he should open a new event to force it to another calendar]

Added a note about that as discussed

jovirkku commented:
Thanks for the "Added a note about that as discussed"

>

jovirkku commented:
I was supposed to say that this is a good document now. Ready to be approved.
